src/lexers/vue-lexer.js

```diff
@@ -0,0 +1,25 @@
+import BaseLexer from './base-lexer'
+import JavascriptLexer from './javascript-lexer.js'
+
+export default class VueLexer extends BaseLexer {
+  constructor(options = {}) {
+    super(options)
+
+    this.functions = options.functions || ['$t']
+  }
+
+  extract(content, filename) {
+    let keys = []
+
+    const Lexer = new JavascriptLexer()
+    Lexer.on('warning', warning => this.emit('warning', warning))
+    keys = keys.concat(Lexer.extract(content))
+
+    const compiledTemplate = require('vue-template-compiler').compile(content).render
```

Comment:
I'm not familiar with the Vue compiler but do we need to run 2 lexers? Couldn't we do it once only?

Reply:
vue compiler return jsx-like template separately. then I run the result through js-lexer.